CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 
 ### Snowpark pandas API Updates
 
+### Dependency Updates
+
+- Updated `modin` from 0.28.1 to 0.30.1.
+- Added support for `pandas` 2.2.2 and 2.2.3.
```

Comment:
you mean updated pandas support from 2.2.1 to 2.2.*, right?

Reply:
yes, I'll update this to clarify